apps/gp2-frontend/src/outputs/state.ts

```diff
@@ -34,3 +35,25 @@ export const useCreateOutput = () => {
     return output;
   };
 };
+
+export const useAuthorSuggestions = () => {
+  const authorization = useRecoilValue(authorizationState);
+
+  return async (searchQuery: string) => {
+    const users = await getUsers(
+      { search: searchQuery, skip: 0, take: 10 },
+      authorization,
+    );
+    const externalUsers = await getExternalUsers(
+      { search: searchQuery, skip: 0, take: 10 },
+      authorization,
+    );
+    return [...users.items, ...externalUsers.items]
+      .map((author) => ({
+        author,
```

Comment:
this will create a property called author, with the author data in it. Should this be removed, or spread inside the object instead?